tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -6177,3 +6178,67 @@ def test_user_display(self):
         assert len(data) == 1
         result = {r["user.display"] for r in data}
         assert result == {"hellboy@bar.com"}
+
+    def test_all_events_fields(self):
```

Comment:
this test is our new #1 flakiest test -- can you disable it and/or fix it?

https://sentry.sentry.io/issues/4682825786/?project=2423079&query=is%3Aunresolved&referrer=issue-stream&sort=freq&statsPeriod=24h&stream_index=1